specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21879,12 +21879,12 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflowConfigurations/{configurationName}": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflowsConnections": {
+      "post": {
         "tags": [
           "WebApps"
         ],
-        "summary": "Get workflow app's configurations connections information by its ID for web site, or a deployment slot.",
+        "summary": "Lists workflow app's configurations connections information for web site, or a deployment slot.",
         "operationId": "WebApps_ListInstanceWorkflowsConfigurationConnectionsSlot",
```

Comment:
Should probably be more descriptive of what's actually happening. 
instead of WebApps_ListInstanceWorkflowsConfigurationConnectionsSlot 
change to "WebApps_ListWorkflowsConnectionsSlot"

Reply:
Updated here and elsewhere.